### PR TITLE
add new theme description for solo-theme-bundle

### PR DESCRIPTION
**Pull request by denniserdmann**

### CODE REVIEW

**Review by xchs — APPROVED** [review]

**Comment by denniserdmann** [comment]
@xchs since my pr was approved and all checks passed, when will my changes be merged?

**Comment by xchs** [comment]
> @xchs since my pr was approved and all checks passed, when will my changes be merged?

@denniserdmann This depends on @richardhj You requested a review from him (which has not been done yet). You can wait until he approves it too. Or you remove the review request from him and I'll merge it right away.

**Review by richardhj — APPROVED** [review]

**Comment by denniserdmann** [comment]
> You requested a review from him

I didn't know i did that. Maybe GitHub remembers pull requests in the past? Nevertheless thanks for merging!

**Pull request merged**
